### PR TITLE
use docker hbase for tests and refactor

### PR DESCRIPTION
**Pull request by shekenahglory**

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -18,6 +44,24 @@ function CrawlHbaseClient(dbUrl) {
...
+CrawlHbaseClient.prototype.initTables = function(recreate) {
+  var self = this;
+
+  if (recreate) {
+    return Promise.map(tableList, function(table) {
+      return self._hbase.deleteTable(table);
+    }).then(createTables);
+
+  } else {
+    return craeteTables();
```
Comment (wilsonianb) [inline src/client.js]: `craeteTables` -> `createTables`

**Comment by wilsonianb** [comment]
Should `src/hbase-thrift/index.js` be its own npm module since we are re-using it multiple projects?
Alternatively, should crawler stuff be moved to rippled-historical-database?

**Comment by shekenahglory** [comment]
yes to both, probably.  I do want to reformat the crawler data at some point to be more consistent with the other table we are storing.  We probably need to think about how all of these services should be organized, and do some refactoring.

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**